benchmarks/python/core.py

```diff
@@ -170,7 +171,9 @@ class NVFBenchmark:
     torchprofiler-based timer and metric computation.
     """
 
-    def __init__(self, benchmark_fixture, precision: float = 1e-6):
+    def __init__(
+        self, benchmark_fixture, device: str = "cuda", precision: float = 1e-6
+    ):
```

Comment:
Nit: please add description for device argument to docstring. In particular, what values are expected?

Reply:
Also, I think calling this argument `device` is confusing. The test will still run on `cuda` device irrespective of this argument. Instead, I think it would be clearer to have something like `measure_execution_only: bool = True` or `measure_compile_time: bool = False`.

Reply:
Will add a description. I like the idea of using `measure_compile_time` instead of `device`.
Detailed comments are in `run_benchmark` since that is the user-facing function.